index.js

```diff
@@ -58,3 +59,22 @@ const show = (element, link) => {
     }
   });
 });
+
+// add date
+let suffix;
+const date = DateTime.now().toFormat('d');
+
+if (+date[1] === 1 && +date !== 11) {
+  suffix = 'st';
+} else if (+date[1] === 2 && +date !== 12) {
+  suffix = 'nd';
+} else if (+date[1] === 3 && +date !== 13) {
+  suffix = 'rd';
+} else {
+  suffix = 'th';
+}
+
+setInterval(() => {
+  const now = DateTime.now().toFormat(`MMMM dd'${suffix}' yyyy', ' hh:mm:ss a`);
+  document.getElementById('date').innerHTML = now;
+}, 1000);
```

Comment:
- [ ] It will be great to have this as a module in your application. Remember we want to practice the creation of different modules in your application. Since this code is responsible for displaying the date you can have it as a function in a separate module say `date module` then import it to this file and call the function. The more we separate our code into modules with different concerns the better. This will improve our code quality

Reply:
Fixed.